src/Factory.php

```diff
@@ -24,14 +24,17 @@
     public static function create()
     {
         // @codeCoverageIgnoreStart
-        if (\class_exists('libev\EventLoop', false)) {
+        if (\function_exists('uv_loop_new') && \PHP_MAJOR_VERSION === 7) {
+            // only use ext-uv on PHP 7
+            return new ExtUvLoop();
```

Comment:
Is the version check needed here? I would vote to remove this, the extension is only available on PHP 7+ anyway, so this is not a decision we need to make.